### PR TITLE
Send email

### PR DESCRIPTION
**Pull request by jcolairo**
- got order to format properly in email
- added multiple products to an order
- sent email to client and owner

### CODE REVIEW

**Comment by jcolairo** [comment]
we havent got the total price for the order in the email

File: api/utilities/emailManager.js [hunk 1]
```diff
@@ -1,14 +1,85 @@
...
+    path: '/v3/mail/send',
+    body: mail.toJSON()
+  });
 
+  sg.API(request, function (error, response) {
+    if (error) {
+      console.log('Error response received');
+    }
+    console.log(response.statusCode);
+    console.log(response.body);
```
Comment (CrocoNiall) [inline api/utilities/emailManager.js]: Do you need these console logs?

File: api/utilities/emailManager.js [hunk 2]
```diff
@@ -1,14 +1,85 @@
...
+    path: '/v3/mail/send',
+    body: mail.toJSON()
+  });
+
+  sg.API(request, function (error, response) {
+    if (error) {
+      console.log('Error response received');
+    }
+    console.log(response.statusCode);
+    console.log(response.body);
```
Comment (CrocoNiall) [inline api/utilities/emailManager.js]: console logs?

**Comment by CrocoNiall** [comment]
I would put all require statements at the top of the file, outside of the function. This is just for readability and module caching

**Comment by CrocoNiall** [comment]
Also, remove comments

File: api/utilities/emailManager.js (hunk 2 above)
Comment (jcolairo) [reply api/utilities/emailManager.js]: these logs came with the code that sendgrid gave me. Do you think i should get rid of them?

**Comment by jcolairo** [comment]
also the required statements are split up because the emails are sent to the user and the admin. would it still send to both emails if the statements are together?

**Comment by CrocoNiall** [comment]
Looks good.

**Pull request merged**